src/olympia/addons/models.py

```diff
@@ -1720,21 +1720,27 @@ class AddonApprovalsCounter(ModelBase):
     addon = models.OneToOneField(
         Addon, primary_key=True, on_delete=models.CASCADE)
     counter = models.PositiveIntegerField(default=0)
+    last_human_review = models.DateTimeField(null=True)
 
     def __unicode__(self):
         return u'%s: %d' % (unicode(self.pk), self.counter) if self.pk else u''
 
     @classmethod
     def increment_for_addon(cls, addon):
         """
-        Increment approval counter for the specified addon. If an
-        AddonApprovalsCounter already exists, it updates it, otherwise it
-        creates and saves a new instance.
+        Increment approval counter for the specified addon, setting the last
+        human review date to now. If an AddonApprovalsCounter already exists,
+        it updates it, otherwise it creates and saves a new instance.
         """
+        data = {
+            'counter': 1,
+            'last_human_review': datetime.now(),
```

Comment:
😭 but given that we do store dates like that everywhere right now it's perfect the way it is. Still… 😭